dask/dataframe/core.py

```diff
@@ -579,6 +579,17 @@ def _aca_agg(self, token, func, aggfunc=None, **kwargs):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
+    def apply_concat_apply(self, func, aggfunc=None, func_kwargs=None,
+                           aggfunc_kwargs=None):
+        """ Apply a function to each partition, the concat, then apply again
+        """
+        token = '-{}-{}'.format('apply_concat_apply',
```

Comment:
Many names use hyphens rather than underscores, so perhaps `apply-concat-apply`. Alternatively maybe just let `_aca_agg` determine its own token from the function names?  It'd be nicer to have names like `sum` that are based on the function names instead of `apply-concat-apply` which is somewhat less informative.  These names end up in diagnostics like the progressbar